source/f4_testing/main.c

```diff
@@ -63,6 +64,11 @@ GPIOInitConfig_t gpio_config[] = {
     GPIO_INIT_AF(SPI_MOSI_PORT, SPI_MOSI_PIN, 5, GPIO_OUTPUT_HIGH_SPEED, GPIO_OUTPUT_PUSH_PULL, GPIO_INPUT_PULL_DOWN),
     GPIO_INIT_AF(SPI_MISO_PORT, SPI_MISO_PIN, 5, GPIO_OUTPUT_HIGH_SPEED, GPIO_OUTPUT_OPEN_DRAIN, GPIO_INPUT_OPEN_DRAIN),
 
+    // CAN
+    // FIXME: (I think these are backwards on schematic? Or are the defs different?)
```

Comment:
Wdym here - I'm peeking at the mcu schematic and VCAN matches this def here with PA11 on RX and PA12 on TX.

Reply:
I think I may have just been a bit silly. I am unsure lol.